src/guide/contributing-to-an-open-jdk-project.md

```diff
@@ -8,6 +8,12 @@ If you have a success story where Java solved your problem, or if you successful
 
 If you're in a position to choose what programming language to use in a project, in a tutorial, or in a class, you have the power to enlarge the Java community in a very direct way, and your colleagues or students will get an opportunity to learn one of the most used programming languages in the world.
 
+## Requesting a change
+
+If you want to request a change to the JDK, but don't have the intention to implement/contribute to the change yourself, you should always go through [bugreport.java.com](bugreport.java.com). This is also the case if you find a change that you would like to see backported to an eralier JDK release. **Never** reach out directly to any developer. Also, please note that the [OpenJDK mailing lists](#mailing-lists) are intended for developers and contributors. These lists are not support channels or channels for requesting changes.
+
+Before you reach out, make sure you have read [Things to consider before proposing changes to OpenJDK code](#things-to-consider-before-proposing-changes-to-openjdk-code) and [Why is my change rejected?](#why-is-my-change-rejected) below.
```

Comment:
In light of your revision in the previous sentence, perhaps "Before you reach out" -> "Before you file an issue"?